internal/verification/model/health_authority.go

```diff
@@ -79,6 +80,15 @@ func (k *HealthAuthorityKey) IsValidAt(t time.Time) bool {
 	return t.After(k.From) && (k.Thru.IsZero() || k.Thru.After(t))
 }
 
+// Revoke revokes a key.
+func (k HealthAuthorityKey) Revoke() {
+	k.Thru = time.Now()
```

Comment:
Do we need to UTC this?

Reply:
This was just cargo-culted from the old code. Do we need to do the same for all other times? Eg, the `From` field (which is _not_ currently like that)...

Reply:
so, technically, no. The Go comparisons will take time zone into account. But we attempt to make all of our times UTC for storage.